stock_picking_report_valued_copy_unvalued/views/stock_picking_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Alberto Martínez <alberto.martinez@sygel.es>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="view_picking_form_unvalued_copy" model="ir.ui.view">
+        <field name="name">stock.picking.form.unvalued.copy</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='other_infos']" position="inside">
+                <field name="valued" invisible="1" />
+                <field name="unvalued_copy" invisible="not valued" />
```

Comment:
Shouldn't it be hidden if it's not of the outgoing type? Mainly because the report takes it into account and it could cause confusion.